pkg/client/factory.go

```diff
@@ -79,14 +79,12 @@ func (f *OptimizelyFactory) Client(clientOptions ...OptionFunc) (*OptimizelyClie
 
 	eg := utils.NewExecGroup(ctx, logging.GetLogger(f.SDKKey, "ExecGroup"))
 	appClient := &OptimizelyClient{execGroup: eg,
-		notificationCenter:   registry.GetNotificationCenter(f.SDKKey),
-		logger:               logging.GetLogger(f.SDKKey, "OptimizelyClient"),
-		defaultDecideOptions: []decide.Options{},
+		notificationCenter: registry.GetNotificationCenter(f.SDKKey),
+		logger:             logging.GetLogger(f.SDKKey, "OptimizelyClient"),
 	}
 
-	if len(f.defaultDecideOptions) > 0 {
-		appClient.defaultDecideOptions = f.defaultDecideOptions
-	}
+	copiedOptions := f.defaultDecideOptions
```

Comment:
why are we doing this?

Reply:
removing this since its not needed here, thanks for pointing out 👍